frontend/src/common/constansts/titles.ts

```diff
@@ -3,13 +3,16 @@ export enum Titles {
     QR_GENERATION = "QR Generation",
     TRANSACTIONS = "Transactions",
     HOME = "Home",
-    SIGN_UP = "Sign Up",
     SIGN_IN = "Sign In",
     PAYMENT = "Payment",
     PROFILE = "Profile",
     TOP_UP = "Top up",
     LOG_OUT = "Log out",
     SETTINGS_HINT = "Open settings",
     SELECT_ROLES = "Select role",
-    LOG_OUT_DIALOG = "Log out"
+    LOG_OUT_DIALOG = "Log out",
+    BALANCE = "Balance",
+    SCAN_PAY = "Scan & Pay",
+    GENERATE_QR = "Generate QR code",
+    ABOUT_QR = "QR code will be displayed here"
 }
```

Comment:
To reduce confusion, I would use this titles enum for actual titles: Transactions, QR Generation, Home, Profile . But any other string move to a separate enum, maybe to call it Text enum ?